server/schema/user/user.resolver.js

```diff
@@ -43,9 +43,7 @@ const canUpdateUser = (id, mid, session, authToken, decodedToken, fieldNodes, ne
   if (needsAdmin && !UserPermission.exists(session, authToken, decodedToken, 'user.write.all')) {
     throw APIError('FORBIDDEN', null, { reason: 'The user does not have the required administrative priveledges.' });
   } else if (
-    (mid === id &&
-      !UserPermission.exists(session, authToken, decodedToken, 'user.write.self') &&
-      !UserPermission.exists(session, authToken, decodedToken, 'user.write.all')) ||
+    (mid === id && !UserPermission.exists(session, authToken, decodedToken, 'user.write.self')) ||
```

Comment:
Why this change?

Reply:
This basically checking the user can updates some thing from own account or not. user.write.self is sufficient  to do that. And super admin always have that role.